lib/modules/datasource/custom/schema.ts

```diff
@@ -2,14 +2,23 @@ import { z } from 'zod';
 
 export const ReleaseResultZodSchema = z.object({
   releases: z.array(
-    z.object({
-      version: z.string(),
-      isDeprecated: z.boolean().optional(),
-      releaseTimestamp: z.string().optional(),
-      sourceUrl: z.string().optional(),
-      sourceDirectory: z.string().optional(),
-      changelogUrl: z.string().optional(),
-    }),
+    z
+      .object({
+        version: z.string(),
+        isDeprecated: z.boolean().optional(),
+        releaseTimestamp: z.string().optional(),
+        sourceUrl: z.string().optional(),
+        sourceDirectory: z.string().optional(),
+        changelogUrl: z.string().optional(),
+        digest: z.string().optional(),
```

Comment:
```suggestion
        newDigest: z.string().optional(),
```

Can the transform step be skipped if the above is in place?

Reply:
Yes, it can. Also there will be nothing to test then? I've just followed the naming from the original issue (and, subjectively, would prefer "digest")

Also, maybe there's an less ugly way of doing the transform - I just have no experience with TypeScript.

Reply:
I think I understand what you mean, and I prefer `digest` too. We should probably make that change wider (but not in this PR)